taskana-adapter-outbox-rest-spring-boot-starter/pom.xml

```diff
@@ -39,6 +39,12 @@
                         <version>${version.postgres}</version>
                         <scope>test</scope>
                 </dependency>
+				<dependency>
```

Comment:
This looks like your formatter works different to the other formatters?

Reply:
Very strange, I use the taskana QA IntelliJ Formatter...it doesn't show it like that in my IDE

Reply:
They do on xml files. Don't worry about it. Will be fixed over the holidays :)